src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -7920,22 +8413,25 @@ namespace HVACVariableRefrigerantFlow {
                 // Air terminal supply side mixer, calculate supply side mixer output
                 SimATMixer(this->ATMixerName, FirstHVACIteration, this->ATMixerIndex);
                 TempOut = DataLoopNode::Node(ATMixOutNode).Temp;
-                TempIn = DataLoopNode::Node(ZoneNode).Temp;
                 SpecHumOut = DataLoopNode::Node(ATMixOutNode).HumRat;
-                SpecHumIn = DataLoopNode::Node(ZoneNode).HumRat;
-                AirMassFlow = DataLoopNode::Node( ATMixOutNode ).MassFlowRate;
+                AirMassFlow = DataLoopNode::Node(ATMixOutNode).MassFlowRate;
             } else {
                 // Air terminal inlet side mixer
                 TempOut = DataLoopNode::Node(VRFTUOutletNodeNum).Temp;
-                TempIn = DataLoopNode::Node(ZoneNode).Temp;
                 SpecHumOut = DataLoopNode::Node(VRFTUOutletNodeNum).HumRat;
-                SpecHumIn = DataLoopNode::Node(ZoneNode).HumRat;
             }
```

Comment:
Just moved these next 2 lines from being duplicate withing the IF/ELSE block above.